packages/vite/src/client/overlay.ts

```diff
@@ -201,10 +215,6 @@ export class ErrorOverlay extends HTMLElement {
       }
     }
   }
-
-  close(): void {
-    this.parentNode?.removeChild(this)
-  }
```

Comment:
I don't think we can remove this, or it'll break:

https://github.com/vitejs/vite/blob/ec9d2e779d4b8d785c648430594d534d461d6639/packages/vite/src/client/client.ts#L297-L301